src/Microsoft.AspNet.Cors/CorsMiddleware.cs

```diff
@@ -4,7 +4,7 @@
 using System;
 using System.Threading.Tasks;
 using Microsoft.AspNet.Builder;
-using Microsoft.AspNet.Cors.Core;
+using Microsoft.AspNet.Cors.Infrastructure;
 using Microsoft.AspNet.Http;
 using Microsoft.Extensions.Primitives;
 
```

Comment:
Yeah everything except the two attributes I think. It's in https://github.com/aspnet/CORS/issues/41.